### PR TITLE
feat(addon-mobile): possibility drop mobile calendar header

### PR DESCRIPTION
**Pull request by MillerSvt**
Closes #

### CODE REVIEW

File: projects/i18n/types/language.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export interface TuiLanguageKit {
     /**
      * [@string 'choose day', @param 'choose range', @param 'choose days']
      */
-    mobileCalendarTexts: [string, string, string];
+    mobileCalendarTexts: [string, string, string] | null;
```
Comment (nsbarsukov) [inline projects/i18n/types/language.ts:64]: The same comments

https://github.com/taiga-family/taiga-ui/pull/8264#discussion_r1696704986

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/mobile-calendar/examples/6/index.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="example">
+    <tui-mobile-calendar
+        [max]="max"
+        [min]="min"
+    />
+</div>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/mobile-calendar/examples/6/index.html:6]: The same comment

https://github.com/taiga-family/taiga-ui/pull/8264#discussion_r1698125359

**Comment by splincode** [comment]
@MillerSvt any updates?

**Comment by MillerSvt** [comment]
@splincode awaiting #8296

**Comment by splincode** [comment]
@MillerSvt merged

**Comment by splincode** [comment]
@MillerSvt why cypress tests are failed?

**Comment by MillerSvt** [comment]
@splincode I think test is incorrect.
1. In Test component, we provide `TUI_CALENDAR_DATE_STREAM` with `TuiDay` of tomorrow
2. When we create component, tomorrow date will automatically select
3. When we click on today date, `TuiDayRange` should be `TuiDayRange(today, tomorrow)` (not `TuiDayRange(today, today)`)

What do you think?

**Comment by splincode** [comment]
@MillerSvt can you rewrite the test?

**Comment by splincode** [comment]
@nsbarsukov

**Pull request merged**